python_modules/dagster/dagster/_core/definitions/partitions/definition/time_window.py

```diff
@@ -653,8 +728,10 @@ def _get_last_partition_window(self, current_timestamp: float) -> Optional[TimeW
         else:
             return current_timestamp_window
 
-    def get_last_partition_window(self) -> Optional[TimeWindow]:
-        return self._get_last_partition_window(self._get_current_timestamp())
+    def get_last_partition_window(self, ignore_exclusions: bool = False) -> Optional[TimeWindow]:
```

Comment:
would prefer to avoid adding `ignore_exclusions` to this more "public" function (it's not public, but it's definitely more likely to be found / used by an enterprising user) and just use the underscored version if possible

given this is just for the TimeWindowPartitionsDef, maybe we just have a separate function `get_last_partition_window_without_exclusions`?

Reply:
good idea